src/common/misc/LazySearchBar.ts

```diff
@@ -9,7 +10,8 @@ import { LazyLoaded } from "@tutao/tutanota-utils"
  */
 export class LazySearchBar implements ClassComponent<SearchBarAttrs> {
 	private static searchBar: LazyLoaded<SearchBar> = new LazyLoaded(async () => {
-		const { searchBar } = await import("../../mail-app/search/SearchBar.js")
+		const searchbarPath = client.isCalendarApp() ? "../../calendar-app/search/CalendarSearchBar.js" : "../../mail-app/search/SearchBar.js"
```

Comment:
if `IS_CALENDAR_APP` would be a constant it would be possible to import just one but this way I think it will bundle both?

Reply:
Good catch! We just did a test, and it seems that Rollup managed to figure this bundling out already and only includes the needed file.